.github/ISSUE_TEMPLATE/---report-an-issue.md

```diff
@@ -1,21 +1,24 @@
 ---
 name: "\U0001F41B Report an issue"
 about: Report an issue on parse-server
+title: ''
+labels: ''
+assignees: ''
 
 ---
 
-<!--- 
+We use GitHub Issues for reporting bugs with Parse Server.
 
-**We use GitHub Issues for reporting bugs with parse-server.**
+Make sure these boxes are checked before submitting your issue - thanks for reporting issues back to Parse Server!
 
-If you have a *question*, you should join the [Parse Community's Discourse forum](https://community.parseplatform.org/c/parse-server).
+- [ ] This isn't a vulnerability disclosure, if it is please follow our [security policy](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
 
-If you have a vulnerability disclosure, please follow our policy available here https://github.com/parse-community/parse-server/blob/master/SECURITY.md
+- [ ] You're running version >=2.3.2 of Parse Server, we can't accept issues for very outdated releases, please update to a newer version.
 
-You may also search through existing issues before opening a new one: https://github.com/parse-community/parse-server/issues?utf8=%E2%9C%93&q=is%3Aissue 
+- [ ] This isn't a question, code level questions should be posted on Stack Overflow with the [parse-platform tag](https://stackoverflow.com/questions/tagged/parse-platform) & other questions can be posted on the [community forum](https://community.parseplatform.org).
```

Comment:
I wonder whether "code level question" is a term that is widely understood by specifically those authors, who usually post code level questions, who sometimes seem to be novice programmers? Do you think it helps to use more general terms to make it clear like "troubleshooting, general help, coding help"?

Reply:
Fair point, I’ve adjusted it